src/ifElse.test.js

```diff
@@ -1,11 +1,27 @@
 'use strict';
 
 describe('ifElse', () => {
-  // const { ifElse } = require('./ifElse');
+  const { ifElse } = require('./ifElse');
 
-  it('should ', () => {
+  it('should return `first`', () => {
```

Comment:
It should not return `first` or `second`, it should call one of these functions.
Also, add a condition in test description, when exactly `first` or `second` should be called